examples/tts/speechllm/conf/megatron_t5_speechllm_inference.yaml

```diff
@@ -63,7 +63,12 @@ model:
   save_nemo_on_validation_end: True # Saves an inference ready .nemo file every time a checkpoint is saved during training.
   existing_tasks: []
   new_tasks: ["squad"]
-
+  codecmodel_type: nemo_codec
+  codecmodel_path: ???
+  english_only_model: true
+  context_conditioning: decoder
+  use_flash_attention: false
```

Comment:
This is true in the other inference file, we should agree on one

Reply:
Sure, let's leave it as false? (Since pretrained model and interspeech checkpoitns are all false I believe)